docs/semgrep-assistant/overview.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: overview
+append_help_link: true
+title: AI-assisted triage and autofix
+hide_title: true
+description: Learn how Semgrep Assistant can provide recommendations for triage and remediation of Semgrep findings. 
+tags:
+  - Semgrep Code
+  - Semgrep Supply Chain
+  - Semgrep Secrets
+  - Semgrep Cloud Platform
+  - Semgrep Assistant
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Semgrep Assistant overview
+
+Semgrep Assistant provides GPT-4-powered security recommendations to help you review, triage, and remediate your Semgrep findings.
+
+![Screenshot of Semgrep Assistant message in GitHub](/img/semgrep-assistant-github.png)
+*Figure*. Semgrep Assistant detects a false positive.
+
+## Support and availability
+
+Semgrep Assistant supports findings generated by Semgrep Code, Semgrep Supply Chain, and Semgrep Secrets, and, in many cases, requires using Semgrep Cloud Platform. It is not available on the Semgrep CLI.
+
+Semgrep Assistant is available to users of the following source code managers (SCMs):
+  * GitHub Cloud
+  * GitLab, including SaaS and self-managed plans
+
+Semgrep Assistant does not support the use of GitHub Enterprise Server (self-hosted).
+
+### Language support
+
+Semgrep Assistant supports [the same languages as Semgrep Code](/supported-languages).
+
+## Features
+
+### Component tags
+
+**Component tags** use GPT-4 to categorize a finding based on its function, such as:
+
+- Payments
+- User authentication
+- Infrastructure
+
+By categorizing your code through component tags, Semgrep Assistant can help you prioritize **high-risk issues**, such as remediating a code finding related to payments or user authentication.
+
+![Semgrep Assistant Component tag list](/img/assistant-component-tags.png)
+
+### Rules recommendations
```

Comment:
```suggestion
### Suggested categories for rules
```

I tried to come up with an unambiguous title.

"Rules recommendations" could be "giving users recommendations for rules to add in Policies" and "Categorizing rules" could be implying the AI does this automatically once enabled. So I went with "suggest".